source/configuration_manual/fts/squat.rst

```diff
@@ -3,11 +3,9 @@
 Squat FTS Engine
 ================
 
-.. deprecated:: v2.1
+.. versionremoved:: v3.0
```

Comment:
I wonder if we should leave both. Not sure..

Reply:
I wondered the same, but have found no cases of use of `versionremoved` where`deprecated` has been preserved.